src/app.jsx

```diff
@@ -311,6 +311,9 @@ class Application extends React.Component {
                         reply = reply[0];
 
                         reply.isSystem = system;
+                        // HACK: during restart State never changes from "running"
```

Comment:
if we would change this to:
```
+                        if (event.Action === "restart")
+                            reply.State = "restarting";
```
we don't need any additional changes and it will also update the label. We should then only define it in `src/utils.js` for it being localized.

WDYT?

Reply:
That was my first idea but it has some unwanted side effects. See: https://github.com/cockpit-project/cockpit-podman/issues/1089#issuecomment-1240676131

Reply:
Ah right. So we switched by default to showing all containers and not only running ones, so this should not be a big deal. Also we could show restarting as running

Reply:
Hmmm. I think changing the filter to include "restarting" even when "only running" is selected might be a good/better solution.

Reply:
done

Reply:
Please add the `restarting` translation to `src/utils.js` with a comment why it's explicitly added there.

Reply:
Hi, the `restarting` translation has been added to `src/utils.js` by @skazi0